packages/geolocator/tizen/src/locaton_manager.h

```diff
@@ -12,8 +12,14 @@
 #include "location.h"
 #include "tizen_result.h"
 
+// Defined in:
+
+// https://github.com/Baseflow/flutter-geolocator/blob/master/geolocator_platform_interface/lib/src/enums/location_service.dart
```

Comment:
Unexpected newline between the comments.

Reply:
Thanks!